src/test/java/eu/dissco/backend/repository/ElasticSearchRepositoryIT.java

```diff
@@ -54,7 +61,7 @@ class ElasticSearchRepositoryIT {
   private static final String ELASTICSEARCH_PASSWORD = "s3cret";
   private static final String CREATED_ALT = "2022-09-02T09:59:24Z";
   private static final ElasticsearchContainer container = new ElasticsearchContainer(
-      ELASTIC_IMAGE).withExposedPorts(9200).withPassword(ELASTICSEARCH_PASSWORD);
+      ELASTIC_IMAGE).withPassword(ELASTICSEARCH_PASSWORD);
```

Comment:
Why remove withExposedPorts()?

Reply:
It's not necessary to expose this ports and when you run it while you have an instance of elastic running locally on 9200 it will conflict and be unable to run. Testcontainer normally just picks a random port in the (30XXX range I believe) which are generally always free. That way we don't get unnecessary port conflicts.

Reply:
Not in the tests themselves, but if you need to debug something later on it might be nice to know which port the testcontainer is running on. Otherwise, it'll just run on a random port. Up to you